### PR TITLE
adds documentation for shipctl get_git_changes

### PR DESCRIPTION
**Pull request by scriptnull**
https://github.com/Shippable/docs/issues/1196

image

### CODE REVIEW

File: sources/platform/tutorial/workflow/using-shipctl.md [hunk 1]
```diff
@@ -661,6 +661,19 @@ shipctl decrypt <source filename> <key filename>
...
+```
+shipctl get_git_changes <--path|--resource>
+```
+
+- `--path` is the file system path to a git repository.
+- `--resource` is the name of the [gitRepo resource](/platform/workflow/resource/gitrepo/).
+- `--commit-range` specifies the range of commits to look for changes (Example: `HEAD~1..HEAD`). Defaults to `$SHIPPABLE_COMMIT_RANGE` for CI jobs. If used in [runSh jobs](/platform/workflow/job/runsh/), the commit range is automatically configured by automatically looking at the version of the [gitRepo resource](/platform/workflow/resource/gitrepo/) mentioned in `--resource` argument.
+- `--directories-only` lists only the directories containing changes.
+- `--depth` returns file/folder at certain depth. Root directory has depth value 1.
+
```
Comment (manishas) [inline sources/platform/tutorial/workflow/using-shipctl.md:676]: Please add an example for this....

**Comment by manishas** [comment]
i added a comment.... please add an example of how to use the new method.

File: sources/platform/tutorial/workflow/using-shipctl.md [hunk 2]
```diff
@@ -661,6 +661,25 @@ shipctl decrypt <source filename> <key filename>
...
+**Example**
+
+```
+CHANGED_MICROSERVICES=$(shipctl get_git_changes --resource="monoRepo-gitRepo" --directories-only)
```
Comment (manishas) [inline sources/platform/tutorial/workflow/using-shipctl.md]: If there are changes to several microservices, how is that reflected in CHANGED_MICROSERVICES?

Comment (scriptnull) [reply sources/platform/tutorial/workflow/using-shipctl.md]: If there are changes to several microservices, the names of all the microservices will be saved line by line in the `CHANGED_MICROSERVICES` variable.

**Pull request merged**